app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    Comment.create(content: params[:content],
```

Comment:
Couldn't you do:

comment = User.find().comments.create(params[:comment])
comment.commentable_id = params...
comment.save

?